airflow/models/taskinstance.py

```diff
@@ -335,30 +335,46 @@ def generate_command(dag_id,
         :type local: bool
         :param pickle_id: If the DAG was serialized to the DB, the ID
             associated with the pickled DAG
-        :type pickle_id: unicode
+        :type pickle_id: str
         :param file_path: path to the file containing the DAG definition
+        :type file_path: str
         :param raw: raw mode (needs more details)
+        :type raw: bool
         :param job_id: job ID (needs more details)
+        :type job_id: str
         :param pool: the Airflow pool that the task should run in
-        :type pool: unicode
+        :type pool: str
```

Comment:
You can also use typing in the docstring.
```suggestion
        :type pool: Optional[str]
```

Reply:
Same for the other optional fields.